301-nested-vms-in-virtual-network/azuredeploy.json

```diff
@@ -0,0 +1,561 @@
+    },
+    "HostNetworkInterface1Name": {
+      "type": "string",
+      "metadata": {
+        "description": "Hyper-V Host Network Interface 1 Name, attached to NAT Subnet"
+      },
+      "defaultValue": "HVHOSTNIC1"
+    },
+    "HostNetworkInterface2Name": {
+      "type": "string",
+      "metadata": {
+        "description": "Hyper-V Host Network Interface 2 Name, attached to Hyper-V LAN Subnet"
+      },
+      "defaultValue": "HVHOSTNIC2"
+    },
+    "HostVirtualMachineName": {
+      "type": "string",
+      "defaultValue": "HVHOST",
+      "maxLength": 15,
+      "metadata": {
+        "description": "Name of Hyper-V Host Virtual Machine, Maximum of 15 characters, use letters and numbers only."
+      }
+    },
+    "HostVirtualMachineSize": {
+      "type": "string",
+      "defaultValue": "Standard_D4s_v3",
+      "allowedValues": [
+        "Standard_D2_v3",
+        "Standard_D4_v3",
+        "Standard_D8_v3",
+        "Standard_D16_v3",
+        "Standard_D32_v3",
+        "Standard_D2s_v3",
+        "Standard_D4s_v3",
+        "Standard_D8s_v3",
+        "Standard_D16s_v3",
+        "Standard_D32s_v3",
+        "Standard_D48_v3",
+        "Standard_D64_v3",
+        "Standard_E2_v3",
+        "Standard_E4_v3",
+        "Standard_E8_v3",
+        "Standard_E16_v3",
+        "Standard_E20_v3",
+        "Standard_E32_v3",
+        "Standard_E48_v3",
+        "Standard_E64_v3",
+        "Standard_D48s_v3",
+        "Standard_D64s_v3",
+        "Standard_E2s_v3",
+        "Standard_E4s_v3",
+        "Standard_E8s_v3",
+        "Standard_E16s_v3",
+        "Standard_E20s_v3",
+        "Standard_E32s_v3",
+        "Standard_E48s_v3",
+        "Standard_E64s_v3"
+      ],
+      "metadata": {
+        "description": "Size of the Host Virtual Machine"
+      }
+    },
+    "HostAdminUsername": {
+      "type": "string",
+      "defaultValue": "HostAdmin",
+      "metadata": {
+        "description": "Admin Username for the Host Virtual Machine"
+      }
+    },
+    "HostAdminPassword": {
+      "type": "secureString",
+      "metadata": {
+        "description": "Admin User Password for the Host Virtual Machine"
+      }
+    }
+  },
+  "variables": {
+    "NATSubnetNSGName": "[concat(parameters('NATSubnetName'), 'NSG')]",
+    "hyperVSubnetNSGName": "[concat(parameters('hyperVSubnetName'), 'NSG')]",
+    "ghostedSubnetNSGName": "[concat(parameters('ghostedSubnetName'), 'NSG')]",
+    "azureVMsSubnetNSGName": "[concat(parameters('azureVMsSubnetName'), 'NSG')]",
+    "azureVMsSubnetUDRName": "[concat(parameters('azureVMsSubnetName'), 'UDR')]",
+    "DSCInstallWindowsFeaturesUri": "[uri(parameters('_artifactsLocation'), concat('dsc/dscinstallwindowsfeatures.zip', parameters('_artifactsLocationSasToken')))]",
+    "HVHostSetupScriptUri": "[uri(parameters('_artifactsLocation'), concat('hvhostsetup.ps1', parameters('_artifactsLocationSasToken')))]"
+  },
+  "resources": [
+    {
+      "condition": true,
```

Comment:
remove condition property (since you have no condition) - here and below

Reply:
Updated